src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/custom.py

```diff
@@ -0,0 +1,51 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.mgmt.billing.models import ErrorResponseException
+from azure.cli.core.util import CLIError
+
+
+def cli_billing_list_invoices(client, generate_url=False):
+    '''List all available invoices of the subscription'''
+    try:
+        if generate_url:
+            invoices = client.list(expand='downloadUrl')
+        else:
+            invoices = client.list()
+        return list(invoices)
+    except ErrorResponseException as ex:
```

Comment:
Also, if you do this, then a lot of the custom methods in this file wouldn't be needed and you can use the SDK directly like for `billing invoice show-latest`.